distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -70,6 +70,10 @@ if [ -f "$DEFAULT" ]; then
 	. "$DEFAULT"
 fi
 
+if [ "$ES_USER" != "elasticsearch" ] || [ "$ES_GROUP" != "elasticsearch" ]; then
```

Comment:
I think it would be better to put this in the pre-install scripts. Is there a reason that that wouldn't work?